pkgs/os-specific/linux/cpupower/gui.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gobject-introspection
+, gtk3
+, hicolor-icon-theme
+, meson
+, ninja
+, pkg-config
+, python3
+, systemd
+, wrapGAppsHook
+}:
+
+let
+  # use pythonEnv, so that patchShebang fixup will choose this interpreter
+  pythonEnv = python3.withPackages(ps: with ps; [
+    pygobject3
+    dbus-python
+  ]);
+in
+
+stdenv.mkDerivation rec {
```

Comment:
please switch to using buildPythonApplication, see my expression for lollypop https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/lollypop/default.nix#L25

Reply:
My initial derivation was indeed using `buildPythonApplication` but it included a lot more explicit settings and manual wrappers. Quite like your lollypop expression.
The current version is based on @jonringer's original which seems to avoid the need for this.

However I'm not entire familiar with how pythonEnv achieves this so I'm happy to adjust and follow whichever is more appropriate.

Reply:
adjusted following your suggestion, now using `buildPythonApplication`.